internal/extensionserver/inferencepool.go

```diff
@@ -310,7 +310,7 @@ func buildHTTPFilterForInferencePool(pool *gwaiev1a2.InferencePool) *extprocv3.E
 			ResponseTrailerMode: extprocv3.ProcessingMode_SEND,
 		},
 		AllowModeOverride: allowModeOverride,
-		MessageTimeout:    durationpb.New(5 * time.Second),
+		MessageTimeout:    durationpb.New(300 * time.Second),
```

Comment:
what about make it configurable instead of using a const value?

Reply:
it is not easier to configure for inference pool, since all configuarable params passed from metadata added from cluster/route level hook, but listener level extproc injected in post translate. 

Let us get this default in, and configure out more general ways to config.